src/amg/azext_amg/_help.py

```diff
@@ -280,3 +280,53 @@
     type: command
     short-summary: delete an api key.
 """
+
+helps['grafana service-account'] = """
+    type: group
+    short-summary: Commands to manage service accounts.
+"""
+
+helps['grafana service-account create'] = """
+    type: command
+    short-summary: create a new service account.
```

Comment:
Recommend adding some examples for those create/update commands.

Reply:
Done